roles/pulp-content/templates/pulp-content.deployment.yaml.j2

```diff
@@ -121,6 +120,15 @@ spec:
           ports:
             - protocol: TCP
               containerPort: 24816
+          readinessProbe:
+            httpGet:
+              path: "{{ pulp_combined_settings.content_path_prefix | default('/pulp/content/') }}"
+              port: 24816
+              scheme: HTTP
+            failureThreshold: 5
+            initialDelaySeconds: 60
+            periodSeconds: 10
+            timeoutSeconds: 5
```

Comment:
in a follow up PR I will get it to use the readyz script above

Reply:
I saw that you let everything ready to use the probe in the `is_content_healthy` function.

Do you know what response pulpcore will return for a GET request on `/pulp/content/`?